packages/terra-search-field/lib/SearchField.scss

```diff
@@ -0,0 +1,46 @@
+@import '~terra-mixins';
+
+.terra-SearchField {
+  display: flex;
```

Comment:
This has been updated.  I also removed the 100% width style since this is no longer a bar and just a field.